about.php

```diff
@@ -1,7 +1,12 @@
+<?php
+require_once "php/git.php";
+?>
+
 <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
 <html>
 <head>
     <title>OpenFlights: About</title>
+    <meta name="version" content="<?php echo(Git::getCurrentCommitID() ?? "unavailable"); ?>">
```

Comment:
echo isn't a function either (like `require_once`) - https://www.php.net/manual/en/function.echo.php#refsect1-function.echo-notes

Reply:
> echo isn't a function either (like `require_once`) - https://www.php.net/manual/en/function.echo.php#refsect1-function.echo-notes

Would you prefer the short form syntax here?

```
<?=(Git::getCurrentCommitID() ?? "unavailable")?>
```

I kind of like the parenthesis in this form because it delimits the expression more clearly.